pedpy/__init__.py

```diff
@@ -23,6 +23,12 @@
     load_walkable_area_from_jupedsim_sqlite,
     load_walkable_area_from_ped_data_archive_hdf5,
 )
+from .methods.acceleration_calculator import (
+    AccelerationCalculation,
+    compute_individual_acceleration,
+    compute_mean_acceleration_per_frame,
+    compute_voronoi_acceleration,
```

Comment:
Please add these also to the `__all__` list

Reply:
done